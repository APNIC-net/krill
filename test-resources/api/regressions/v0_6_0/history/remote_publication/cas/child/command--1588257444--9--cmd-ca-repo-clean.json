{
  "actor": "krill",
  "time": "2020-04-30T14:37:24.169869Z",
  "handle": "child",
  "version": 16,
  "sequence": 9,
  "details": "repo_remove_old",
  "effect": {
    "events": [
      16
    ]
  }
}