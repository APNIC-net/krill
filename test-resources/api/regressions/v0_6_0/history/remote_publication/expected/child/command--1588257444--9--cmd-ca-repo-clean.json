{
  "command": {
    "actor": "krill",
    "time": "2020-04-30T14:37:24.169869Z",
    "handle": "child",
    "version": 16,
    "sequence": 9,
    "details": "repo_remove_old",
    "effect": {
      "events": [
        16
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "child",
        "version": 16,
        "details": {
          "repo_cleaned": {
            "embedded": {
              "base_uri": "rsync://localhost/repo/child/",
              "rpki_notify": "https://localhost:3000/test-rrdp/notification.xml"
            }
          }
        }
      }
    ]
  }
}